cmd/cmd.go

```diff
@@ -20,13 +20,13 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"slices"
 	"strconv"
 	"strings"
 
 	"github.com/google/keep-sorted/keepsorted"
 	flag "github.com/spf13/pflag"
 	"golang.org/x/exp/maps"
```

Comment:
Isn't there also a stdlib maps in 1.21, too?

Reply:
Yes, but it doesn't has the `Keys()`/`Values()` functions. https://pkg.go.dev/maps